src/pages/home/report/ReportActionsView.js

```diff
@@ -314,15 +314,7 @@ function arePropsEqual(oldProps, newProps) {
         return false;
     }
 
-    return (
-        oldProps.report.lastReadTime === newProps.report.lastReadTime &&
-        oldProps.report.reportID === newProps.report.reportID &&
-        oldProps.report.policyID === newProps.report.policyID &&
-        oldProps.report.managerID === newProps.report.managerID &&
-        oldProps.report.lastVisibleActionCreated === newProps.report.lastVisibleActionCreated &&
-        oldProps.report.isOptimisticReport === newProps.report.isOptimisticReport &&
-        _.isEqual(oldProps.report.pendingFields, newProps.report.pendingFields)
-    );
+    return _.isEqual(oldProps.report, newProps.report);
```

Comment:
Sorry, I don't get it. By using `_.isEqual(prevReport, nextReport)` we are already checking for every `report` property like `report.total` or `report.reportID` etcetra.

Now why do you want to add `report.total` explicitly, since it's already covered by `_.isEqual` ?

Reply:
Okay I will remove `_.isEqual` and add the individual comparison 👍 But it's really the same thing, anyways I am on it.

Reply:
Okay I understand that 👍